account_fiscal_position_partner_type/models/account_fiscal_position.py

```diff
@@ -20,7 +20,7 @@ def _default_fiscal_position_type(self):
 
     @api.model
     def search(self, args, offset=0, limit=None, order=None, count=False):
-        if self.env.context.get("fiscal_position_type"):
+        if "fiscal_position_type" in self.env.context:
```

Comment:
This is exactly what I suggested in https://github.com/OCA/account-fiscal-rule/pull/206/files#r656460329, but I analyzed it, and determined that it's OK as it is for only acting if a explicit type is given.

Reply:
We know, Pedro, but we found some functional use cases where it should be specified when empty.

For example, for customers that were already created in the database it was auto-applying (calculating) fiscal position OSS rules in SO & Invoices but this one shouldn't be applied.

Reply:
Please specify whole detailed steps to reproduce the problem to judge.

Reply:
Updated on main comment.